scripts/reindex_articles.coffee

```diff
@@ -15,15 +17,15 @@ switch process.env.NODE_ENV
 # Connect to database
 db = mongojs(process.env.MONGOHQ_URL, ['articles'])
 
-db.articles.find({ }).toArray (err, articles) ->
+db.articles.find({}).toArray (err, articles) ->
   console.log(err) if err
-  indexWorker(articles, 0)
+  console.log(articles.length)
+  async.mapSeries(articles, indexWorker, (err, results) =>
```

Comment:
Does this not risk sending a large number of parallel requests to ES? We want to avoid this. The original code used a 30ms delay between calls to keep the rate down to around 30/s. So we need to be able to rate limit this code too. Admittedly there's not too many articles in the database, but in general indexing bursts can affect ES performance.

Reply:
Good question, it actually makes calls synchronously now because `mapSeries` waits for [this callback](https://github.com/artsy/positron/pull/1305/files#diff-d19ee1f1541fd71f3d7da42fe15aaf11R139) before moving on to the next index.

Reply:
:+1: